sycl/include/sycl/info/device_traits.def

```diff
@@ -187,6 +190,18 @@ __SYCL_PARAM_TRAITS_SPEC(device, opencl_c_version, std::string,
 // Extensions
 __SYCL_PARAM_TRAITS_SPEC(device, sub_group_independent_forward_progress, bool,
                          PI_DEVICE_INFO_SUB_GROUP_INDEPENDENT_FORWARD_PROGRESS)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_srgb, bool,
+                         PI_DEVICE_INFO_IMAGE_SRGB)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_mem_channel, bool,
+                         PI_MEM_PROPERTIES_CHANNEL)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_bfloat16, bool,
+                         PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16)
+
+//Deprecated oneapi/intel extensions
+#ifdef __SYCL_PARAM_TRAITS_ENABLE_DEPRECATION
+#undef __SYCL_TRAITS_DEPRECATED_MESSAGE
+#define __SYCL_TRAITS_DEPRECATED_MESSAGE  __SYCL_DEPRECATED("This information decriptor has been moved to namespace ext::oneapi or ext::intel")
+#endif
```

Comment:
This message is a little too general for my taste, especially since it does not convey that their names have been changed. Any good way of being a little more explicit about the replacements?